test/parallel_api/ranges/std_ranges_test.h

```diff
@@ -38,7 +38,12 @@ static_assert(ONEDPL_HAS_RANGE_ALGORITHMS >= 202409L);
 namespace test_std_ranges
 {
 
-inline constexpr std::size_t big_sz = (1<<25) + 10; //32M
+inline constexpr std::size_t big_sz =
+#if PSTL_USE_DEBUG
+    (1<<18) + 10; //256K
```

Comment:
Actually, we are reducing the test coverage here, because some parallel patterns have code differentiation by data sizes, for example `__parallel_merge`...
I would suggest discussing it offline on one of the next technical meeting.